stk/ops/eltwise_ops.py

```diff
@@ -0,0 +1,23 @@
+from stk.matrix import Matrix
+
+def mul(a, b):
+    """
+    Element-wise multiplication of matrices a and b.
```

Comment:
The formatting on this docstring is a little funky - I try to use Google Python docstring style

https://google.github.io/styleguide/pyguide.html#383-functions-and-methods

Reply:
updated, please lmk if that looks better.